pydra/tasks/fsl/preprocess/fnirt.py

```diff
@@ -353,44 +357,7 @@
 ]
 FNIRT_input_spec = specs.SpecInfo(name="Input", fields=input_fields, bases=(specs.ShellSpec,))
 
-output_fields = [
```

Comment:
Why are you removing `output_fields`?

Reply:
Because everything has been moved to `output_file_template` in the input spec.

Reply:
Got it. Do we still need the `FNIRT_output_spec`, then? It feels like it's unnecessary boilerplate, but I can't remember if it's still required in this case.

Reply:
Do you mean I could define a `ShellCommandTask` without an output spec?

Reply:
Let's see.

Reply:
You were correct, I did not know that. Thanks :+1: